app/sources/AppDelegate.m

```diff
@@ -191,8 +191,9 @@ - (void)buildFontMenu:unused {
 
 - (void)receiveFonts:(NSArray *)fonts {
     NSMenu *menu = [fontMenuItem submenu];
-    [menu removeItemAtIndex:0];
-    NSUInteger itemIndex = 0;
+    NSMenuItem *placeholderMenuItem = fontListPlaceholderMenuItem;
+    NSUInteger itemIndex = [menu indexOfItem:placeholderMenuItem];
+    [menu removeItemAtIndex:itemIndex];
```

Comment:
Instead of looking up the item by index, since you already have a reference to the menu item to remove via the outlet, you should be able to use `[menu removeItem:fontListPlaceholderMenuItem];`

Reply:
Actually, I see that `itemIndex` is used below for insertion point, so never mind. However, is `placeholderMenuItem` necessary? Can't `fontListPlaceholderMenuItem` be used directly?

Reply:
You’re right, I can just use `fontListPlaceholderMenuItem` directly. There could be a readability justifiction, but in this case I think the shorter variable name isn’t worth the line of code.

I will also change `removeItemAtIndex` to `removeItem` to prevent momentary confusion like your first comment. I had that confusion too when I reviewed my diff before publishing. I assume that `removeItem` is slower than `removeItemAtIndex` (the docs don’t say), but I think using `removeItem` is worth it for the code clarity, given that there are only eight menu items to iterate through and this function is only run once.